docs/monitoring.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Monitoring for Migrations
+nav_order: 13
+permalink: /monitoring
+---
+
+
+# Monitoring for Migrations
+{: .no_toc }
+
+Spanner Migration Monitoring for Sharded and Non-Sharded Migrations is now available. The monitoring dahsboard can be accessed through both UI and CLI. Spanner Migration Tool uses the [**Google Cloud Monitoring Service**](https://cloud.google.com/monitoring) to create custom dashboard for each migration.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Sharded Migration
+### UI
+1. The Monitoring Dashboard for each shard along with their corresponding shard id and the Aggregated Monitoring Dashboard link can be found under the **Monitoring Dashboards** section on the Prepare Migration page after all the resources have been generated. 
+2. Along with this a list with shards and their corresponding dahsboards can be found on the Aggregated Monitoring Dashboard itself.
+
+### CLI
+1. The unique name for each dahsboard along with the shard id will be printed on the console.
+2. These dashboards can be accessed through cloud monitoring custom dashboards page.
+3. Aggregated Monitoring Dashbaord name will also be provided.
+### Non-Sharded Migration
+### UI
+The Monitoring Dashboard link can be found under the **Monitoring Dashboards** section on the Prepare Migration page after all the resources have been generated. 
+### CLI
+The unique name for the dahsboard will be printed on the console. This dashboards can be accessed through cloud monitoring custom dashboards page.
+
+## Metrics
+
+### Per Shard or Non-sharded Migration Dashboard
```

Comment:
Instead of `non-sharded` and `sharded` I would suggest just call it - `migration` and `sharded migration`, so that customers dont need to care about the word sharded if they dont already know what it means